gamedb.yaml

```diff
@@ -5069,6 +5069,15 @@
   compat_tool: proton-stable
   launch_options: -force -dx11
 
+- name: "Psychonauts 2"
+  platform: gog
+  id: 1508046796
```

Comment:
Is this the id showing in the URL of the game in the Chimera web app?
https://gogdb.org says this is the package id, and the game id is `1731504234`

Reply:
I'll check. I remember looking at the database and picking the ID for a reason. I don't remember what it was, though :D

Reply:
Sorry for the long wait. The Chimera web app does indeed show `1731504234`. I'll fix it.